### ISSUE TITLE
change password after login

### ISSUE THREAD

**Issue by SinadShan** [SinadShan/PassVault#2]
New feature for users to change password after logging in.

**Comment by saivasanth528** [comment]
Do you mean , you want to implement reset password functionality for a global password?
is your application deployed anywhere ?

**Comment by SinadShan** [comment]
Yes, a user should be able to reset global password after logging in. This is an offline application which stores passwords in the user's machine and is not deployed anywhere.

**Comment by saivasanth528** [comment]
Ok, then how you are going to store all the data like their passwords and everything, I have to configure my local database manually ?? .
Can you explain like how the end-user will use this ?

**Comment by SinadShan** [comment]
The passwords are stored locally in a sqlite database. You can take a look at the app by installing the snap or package.
